site/templates/features.php

```diff
@@ -365,11 +365,11 @@
           ]) ?>
           <?= snippet('features/text', [
             'heading' => 'KirbyTags',
-            'text'    => 'When Markdown just isn\'t enough, add your own KirbyTags to simplify plain text formatting and injection of complex custom components into long-form text.'
+            'text'    => 'When Markdown isn\'t enough, add your own KirbyTags to simplify plain text formatting and to inject complex custom components into long-form text.'
           ]) ?>
           <?= snippet('features/text', [
             'heading' => 'Caching',
-            'text'    => 'Kirby comes with mighty caching on board. Not fitting for your project? Add the cache driver on your choice and reduce page loads to the blink of an eye.'
+            'text'    => 'Kirby comes with mighty caching on board. Not fitting for your project? Add your cache driver of choice and reduce page loads in the blink of an eye.'
```

Comment:
Here I meant that the page load only takes a blink of an eye. Not the implementation of the cache driver. That would be a bit exaggerated.

Reply:
I've never seen the prepositon "to" in combination with "blink of an eye". Maybe "reduce page load massively".

Reply:
How about "reduce page loads significantly"

Reply:
That sounds better.